packages/react-aria-components/docs/DropZone.mdx

```diff
@@ -0,0 +1,362 @@
+  padding: 10px;
+  margin-right: 20px;
+  border-radius: 4px;
+}
+
+.draggable.dragging {
+  opacity: 0.5;
+}
+```
+
+</details>
+
+### Heading
+
+The `heading` slot enables the user to reference the text content to define the dropzone's accessible name. 
+
+```tsx example
+import {Text} from 'react-aria-components';
+
+<DropZone>
+  <Text slot="heading">
+    Drop item here
+  </Text>
+</DropZone>
+```
+
+### File Upload
+
+To upload files, pass `FileTrigger` as a child of `DropZone`.
+
+```tsx example
+import {FileTrigger, Button} from 'react-aria-components';
+
+<DropZone>
+  <FileTrigger>
+    <Button>Select a file</Button>
+  </FileTrigger>
+</DropZone>
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.react-aria-Button {
+  --border-color: var(--spectrum-alias-border-color);
+  --border-color-pressed: var(--spectrum-alias-border-color-down);
+  --border-color-disabled: var(--spectrum-alias-border-color-disabled);
+  --background-color: var(--spectrum-global-color-gray-50);
+  --background-color-pressed: var(--spectrum-global-color-gray-100);
+  --text-color: var(--spectrum-alias-text-color);
+  --text-color-disabled: var(--spectrum-alias-text-color-disabled);
+  --focus-ring-color: slateblue;
+
+  color: var(--text-color);
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  appearance: none;
+  vertical-align: middle;
+  font-size: 1.2rem;
+  text-align: center;
+  margin: 0;
+  outline: none;
+  padding: 4px 12px;
+
+  &[data-pressed] {
+    box-shadow: inset 0 1px 2px rgb(0 0 0 / 0.1);
+    background: var(--background-color-pressed);
+    border-color: var(--border-color-pressed);
+  }
+
+  &[data-focus-visible] {
+    border-color: var(--focus-ring-color);
+    box-shadow: 0 0 0 1px var(--focus-ring-color);
+  }
+
+  &:disabled {
+    border-color: var(--border-color-disabled);
+    color: var(--text-color-disabled);
+  }
+}
+
+@media (forced-colors: active) {
+  .react-aria-Button {
+    forced-color-adjust: none;
+    --border-color: ButtonBorder;
+    --border-color-pressed: ButtonBorder;
+    --border-color-disabled: GrayText;
+    --background-color: ButtonFace;
+    --background-color-pressed: ButtonFace;
+    --text-color: ButtonText;
+    --text-color-disabled: GrayText;
+    --focus-ring-color: Highlight;
+  }
+}
+```
+
+</details>
+
+### Visual feedback
+
+A dropzone can provide visual feedback to the user when a drag hovers over the drop target by passing the `getDropOpertation` function. If a drop target only supports data of specific types (e.g. images, videos, text, etc.), then it should implement getDropOperation and return 'cancel' for types that aren't supported. This will prevent visual feedback indicating that the drop target accepts the dragged data when this is not true. [Read more about getDropOperation.](https://react-spectrum.adobe.com/react-aria/useDrop.html#getdropoperation)
```

Comment:
```suggestion
A dropzone displays visual feedback to the user when a drag hovers over the drop target. If a drop target only supports data of specific types (e.g. images, videos, text, etc.), then it should implement the `getDropOperation` prop and return `'cancel'` for types that aren't supported. This will prevent visual feedback indicating that the drop target accepts the dragged data when this is not true. [Read more about getDropOperation.](https://react-spectrum.adobe.com/react-aria/useDrop.html#getdropoperation)
```

I think eventually we can also add the `acceptedDragTypes` prop to make this a bit easier, but that is out of scope for this PR.

Reply:
oh yeah i believe we have a [follow-up ticket](https://github.com/orgs/adobe/projects/19/views/4?filterQuery=dropzone&pane=issue&itemId=31210918) for that already